src/Http/Livewire/LivewireDatatable.php

```diff
@@ -278,7 +279,7 @@ public function resolveAdditionalSelects($column)
         return $selects->count() > 1
             ? new Expression("CONCAT_WS('" . static::SEPARATOR . "' ," .
                 collect($selects)->map(function ($select) {
-                    return "IF($select, $select, '')";
+                    return "COALESCE($select, NULL)";
```

Comment:
Attention: i am not sure what this line change should do. It does break my existing tables that are using Column::callback(). Could you please elaborate what is the meaning of this line ?

Reply:
This line was previously ```COALESCE($select, '')``` which is needed to properly concatenate the values if one of them is null.

Somebody made a PR that changed it to ```COALESCE($select, NULL)``` which apparently fixed some problem in Postgres, but this broke the concatenation for null values. I tired to change it for an IF statement but that has made things worse.

I am going to reinstate the original line.